pallets/space-follows/tests/src/tests.rs

```diff
@@ -0,0 +1,99 @@
+use frame_support::{assert_noop, assert_ok, dispatch::DispatchError};
+use sp_runtime::traits::Zero;
+
+use pallet_spaces::Error as SpacesError;
+use pallet_permissions::SpacePermission as SP;
+use subsocial_support::{mock_functions::*, ContentError, ModerationError};
+use pallet_space_follows::Error as SpaceFollowsError;
+
+use crate::mock::*;
+use crate::tests_utils::*;
+
+#[test]
+fn follow_space_should_work() {
+    ExtBuilder::build_with_space().execute_with(|| {
+        assert_ok!(_default_follow_space()); // Follow SpaceId 1 by ACCOUNT2
+
+        assert_eq!(
+            SpaceFollows::spaces_followed_by_account(ACCOUNT2),
+            vec![SPACE1]
+        );
+        assert_eq!(
+            SpaceFollows::space_followers(SPACE1),
+            vec![ACCOUNT1, ACCOUNT2]
```

Comment:
We don't auto-follow space by space owner anymore:

```suggestion
            vec![ACCOUNT2]
```